specs/IR-0001/suite/test_spec_ir0001.py

```
@@ -0,0 +1,171 @@
+            self.assertIn(loc, s, f"缺目标市场枚举 {loc}")
+        self.assertIn("三语", s, "缺三语字体表述")
+
+    def test_counts(self):
+        s = read(SPEC)
+        missing = [c for c in self.COUNTS if c not in s]
+        self.assertEqual(missing, [], f"计数锚缺失: {missing}")
+
+    def test_no_vague_terms_in_clauses(self):
+        """模糊禁词镜像（g010 口径）：AC/BEH 条款禁模糊词。"""
+        s = read(SPEC)
+        lines = [l for l in s.splitlines()
```

Ruff (0.16.2)

[error] Ambiguous variable name: `l`

(E741)